config/crd/bases/infrastructure.cluster.x-k8s.io_azuremanagedcontrolplanes.yaml

```diff
@@ -251,6 +251,22 @@ spec:
                   DNS service. It must be within the Kubernetes service address range
                   specified in serviceCidr. Immutable.
                 type: string
+              fleetManager:
+                description: FleetManager is the spec for the fleet this cluster is
+                  a member of.
+                properties:
+                  group:
+                    description: Group is the group this member belongs to for multi-cluster
```

Comment:
is that correct? why is the group under the fleetsManager?

Reply:
This is defined in ASO's fleetsManager spec, so I'm not entirely sure: https://azure.github.io/azure-service-operator/reference/containerservice/v1api20230315preview/#containerservice.azure.com/v1api20230315preview.Fleets_Member_Spec